graphs/kg03.qmd

```diff
@@ -28,7 +28,7 @@ variables <- list(tte = "AVAL", is_event = "is_event", arm = "BMRKR2")
 
 We can produce the basic plot using the `g_km()` function from `tern`.
 
-```{r}
+```{r, fig.width=9, fig.height=6}
```

Comment:
why do we need this? otherwise is too large?

Reply:
It's just nice to have so we can control how the figures are proportioned on the website - otherwise they usually look a bit squished.